Solutions/AWS/Data Connectors/azuredeploy_AWSConnector_API_FunctionApp.json

```diff
@@ -0,0 +1,163 @@
+{
```

Comment:
Hey @tungnt77, Please check the [template  ](https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/DataConnectors/Templates/Connector_REST_API_AzureFunctionApp_template/azuredeploy_DataConnector_API_AzureFunctionApp_template.json) and update azuredeploy file format accordingly.